tests/utils.spec.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable import/no-extraneous-dependencies, no-unused-expressions */
+import test from 'tape-catch';
+import {
+  range,
+  padWithDefault,
+  inTileBounds,
+  cutOffImageBounds
+} from '../src/layers/viv-viewer-layer/utils';
+
+test('range test', t => {
+  const expected = [0, 1, 2];
+  t.deepEqual(range(3), expected, 'Create list for range(3)');
+  t.deepEqual(range(0), [], 'Empty list for range(0)');
+  t.end();
+});
+
+test('padWithDefault test', t => {
+  const expected = [3, 4, 5, 2, 2];
+  t.deepEqual(
+    padWithDefault([3, 4, 5], 2, 2),
+    expected,
+    "Pad with two's to five places."
+  );
+  t.end();
+});
+
+test('inTileBounds test', t => {
+  t.ok(
+    inTileBounds({
+      x: 0,
+      y: 0,
+      z: 0,
+      imageWidth: 10,
+      imageHeight: 10,
+      tileSize: 2,
+      minZoom: -5
+    }),
+    '(0,0,0) tile in bounds for imageWidth: 10, imageHeight: 10, tileSize: 2, minZoom: -5'
```

Comment:
Repeating details in message doesn't seem right. Use variables, or just make shorter?